### DISCUSSION TITLE
Icon Request: icon-saw

### DISCUSSION THREAD

**Discussion by augustjared1** [FortAwesome/Font-Awesome#21230] [Icon Requests]
We are a construction company and this symbol can be used in a variety of ways to signify what stage of construction we are in - e.g. Framing.

![hand-saw-buying-guide-inline-crosscut](https://cloud.githubusercontent.com/assets/7094908/25009125/4a017e20-202b-11e7-8db9-f989b313eb5c.jpg)
![saw](https://cloud.githubusercontent.com/assets/7094908/25009126/4a0ac458-202b-11e7-927b-15bed032988b.jpg)

**Comment by tagliala** [comment]
It is possible that this request is a duplicate, but I'm not able to find the original one

**Comment by augustjared1** [comment]
I searched to see if there was another icon request for a saw but couldn't find one

**Comment by dustcrawler** [comment]
This icon would be very welcome :)

**Comment by danielcongrove** [comment]
+1, wood saw icon would be helpful

**Comment by aethior** [comment]
*Related Unicode Emoji* #222
🪚 [U+1FA9A](https://emojipedia.org/emoji/%F0%9F%AA%9A/) Carpentry Saw

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
